src/codeflare_sdk/cluster/cluster.py

```diff
@@ -505,30 +505,50 @@ def torchx_config(
     def from_k8_cluster_object(
         rc, mcad=True, ingress_domain=None, ingress_options={}, write_to_file=False
     ):
+        config_check()
+        openshift_oauth = False
+        if (
+            rc["metadata"]["annotations"]["sdk.codeflare.dev/local_interactive"]
+            == "True"
+        ):
+            local_interactive = True
+        else:
+            local_interactive = False
+        if "codeflare.dev/oauth" in rc["metadata"]["annotations"]:
+            if rc["metadata"]["annotations"]["codeflare.dev/oauth"] == "True":
+                openshift_oauth = True
+        else:
+            for container in rc["spec"]["headGroupSpec"]["template"]["spec"][
+                "containers"
+            ]:
+                if "oauth-proxy" in container["name"]:
+                    openshift_oauth = True
```

Comment:
Small nit for lines 519 and 525, can you just directly set `openshift_oauth` to the conditional?